tutorials/Multimodal_VQA_Interpret.ipynb

```diff
@@ -230,13 +252,13 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [],
    "source": [
     "vqa_resnet = VQA_Resnet_Model(vqa_net.module.text.embedding.num_embeddings)\n",
     "# `device_ids` contains a list of GPU ids which are used for paralelization supported by `DataParallel`\n",
-    "vqa_resnet = torch.nn.DataParallel(vqa_resnet, device_ids=[0,1])\n",
```

Comment:
Removing fixed number of devices in order to be able to take advantage of all of them.